neovim_gui/gtk_ui.py

```diff
@@ -115,6 +116,8 @@ def start(self, bridge):
         window.connect('scroll-event', self._gtk_scroll)
         window.connect('focus-in-event', self._gtk_focus_in)
         window.connect('focus-out-event', self._gtk_focus_out)
+        window.modify_bg(Gtk.StateType.NORMAL,
```

Comment:
I use titing awesome wm and I always get a grey border on the right side of the window without this setting:
![pynvim_border](https://cloud.githubusercontent.com/assets/513316/14900523/387cfe84-0d91-11e6-8a00-2eecca222332.png)